tools/cldr-code/src/test/java/org/unicode/cldr/json/LdmlConvertRulesTest.java

```diff
@@ -76,6 +76,7 @@ void testSplittableAttributes() {
         jsonSplittableAttrs.add(Pair.of("defaultContent", "locales"));
         jsonSplittableAttrs.add(Pair.of("era", "aliases"));
         jsonSplittableAttrs.add(Pair.of("calendarPreference", "ordering"));
+        jsonSplittableAttrs.add(Pair.of("scriptVariant", "base"));
```

Comment:
I'll check on json soon with this .

Reply:
Thanks! I filed tickets, so no immediate need.